support-lib/wasm/djinni_wasm.cpp

```diff
@@ -162,16 +161,15 @@ EM_JS(void, djinni_init_wasm, (), {
         };
 
         Module.callJsProxyMethod = function(obj, method, ...args) {
-            try {
-                return obj[method].apply(obj, args);
-            } catch (e) {
-                return e;
```

Comment:
This is also somewhat dangerous @LiFengSC because we're catching everything, but only re-throwing in c++ if it's an instanceof Error. If anything else gets thrown, we're returning it as if it was the actual return type, which it wasn't . This seems scary if there are ever cases that throw non-Error types

Reply:
Right.  We should check the type of `e` in JS, if it's not an `Error`, then wrap it in an `Error` first.